internal/sigstore/sign_test.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+package sigstore
+
+import (
+	"testing"
+
+	"github.com/sigstore/cosign/v2/pkg/cosign"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSignSignature(t *testing.T) {
```

Comment:
Please add test inputs for (a) wrong password, (b) failing verification, i.e. missmatching keys.